lib/src/signals/logic.dart

```diff
@@ -701,4 +701,24 @@ class Logic {
     }
     return isLogicIn;
   }
+
+  /// c <= a.selectIndex(b); // order matches closer to array indexing, but requires extension on `List<Logic>`
+  Logic selectFrom(List<Logic> busList, {Logic? defaultValue}) {
+    final c = Logic(name: 'selectFrom', width: busList.first.width);
```

Comment:
Maybe name it `selected` or something instead of `c`?  Also, let's make the name `selected` and `naming: Naming.mergeable` since we don't need to maintain the signal presence in generated SV (new feature in ROHD).

Reply:
Add `naming: Naming.mergeable` to the `Logic()` constructor, you may need to `git pull` to get the changes required for this.  This will help hint signal naming for users of the function.